common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -280,7 +282,8 @@ def get_html(self, view=STUDENT_VIEW):  # lint-amnesty, pylint: disable=argument
         # based on user locale.  This exists to support cases where
         # we leverage a geography specific CDN, like China.
         default_cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get('default')
-        cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get(self.system.user_location, default_cdn_url)
+        user_location = self.system.service(self, 'user').get_current_user().opt_attrs[ATTR_KEY_REQUEST_COUNTRY_CODE]
```

Comment:
Could this be `None`?

Reply:
Documentation says it could be an empty string, but not `None`.  This string is stored in the session in the geioinfo middleware, pulled from [openedx.core.djangoapps.geoinfo.api.country_code_from_ip](https://github.com/edx/edx-platform/blob/b1fa2bdae628d7b8195e4069282f3ee76e12136f/openedx/core/djangoapps/geoinfo/api.py#L18-L19).

Why? I don't see anything in this code that makes `None` a risk?

Reply:
Got it. Don't really remember what I was thinking. :)